pom.xml

```diff
@@ -826,7 +821,7 @@
               <execution>
                 <id>attach-sources</id>
                 <goals>
-                  <goal>jar</goal>
+                  <goal>jar-no-fork</goal>
```

Comment:
Note that this may seem to be the opposite of https://github.com/kohsuke/pom/pull/4 but in this case we are adding the attach mojo from scratch, not trying to interoperate with the release profile inherited from the superpom.